### DISCUSSION TITLE
StrategyAdapter is not defined

### DISCUSSION THREAD

**Discussion by willpakpoy** [loopbackio/loopback-next#8485] [Q&A] [answered]
I'm trying to work through this article, but I can't work out where StrategyAdapter in Step 3 comes from. Am I missing something?

https://loopback.io/doc/en/lb4/Authentication-passport.html

**Comment by achrinza** [comment, marked as answer]
Hi @willpakpoy, `StrategyAdapter` can be imported from `@loopback/authentication-passport` like so:

```typescript
import {StrategyAdapter} from '@loopback/authentication-passport`
```

More information can be found in the API docs: https://loopback.io/doc/en/lb4/apidocs.authentication-passport.strategyadapter.html

I'll open a PR to fix that missing import in the docs.
